core/metamodel/src/main/java/org/apache/isis/core/metamodel/facets/actions/notinservicemenu/derived/NotInServiceMenuFacetDerivedFromDomainServiceFacet.java

```diff
@@ -34,7 +34,7 @@
 
     public NotInServiceMenuFacetDerivedFromDomainServiceFacet(
             final NatureOfService natureOfService, final FacetHolder holder) {
-        super(holder);
+        super(holder, Precedence.DERIVED);
```

Comment:
as per comment below, not sure this is Derived, because it can never be overridden by something more specific.

Reply:
Same here was just trying to preserve prev. behavior. Thanks for pointing out the issue. Guess its save to revert this commit.

Reply:
Probably best to rename the facet here also, get rid of the word "Derived" and replace with something like "Inferred".